plugins/builddep.py

```diff
@@ -119,18 +143,27 @@ def _spec_deps(self, spec_fn):
             err = _("Not all dependencies satisfied")
             raise dnf.exceptions.Error(err)
 
-    def configure(self, args):
-        demands = self.cli.demands
-        demands.available_repos = True
-        demands.resolving = True
-        demands.root_user = True
-        demands.sack_activation = True
-
-    @sink_rpm_logging()
-    def run(self, args):
-        rpm_ts = rpm.TransactionSet()
-        for fn in args:
-            if fn.endswith('.src.rpm') or fn.endswith('.nosrc.rpm'):
-                self._src_deps(rpm_ts, fn)
+    def _enable_source_repos(self):
```

Comment:
move the method [1] from download plugin to lib adn import it, please
[1] https://github.com/rpm-software-management/dnf-plugins-core/blob/master/plugins/download.py#L161